### DISCUSSION TITLE
Upload a document with the user message

### DISCUSSION THREAD

**Discussion by pythiantech** [VRSEN/agency-swarm#27] [Q&A]
First and foremost, thank you so much for creating this!

I am a complete novice and was wondering if there is some way in which I can upload documents to the user input message. My use case is to automate replies to mails received. However a lot of these mails also have PDF or docx attachments. Is there some way I can attach this with the user message?

**Comment by VRSEN** [comment]
Yeah, great suggestion. I will work on this next

**Reply by trees777** [reply]
Awesome!! Thank you Arsenii this will help my previous question as well and would be an incredible add on. I can see so many use cases here.
